FrontEnd/src/UserAuthentication/LoginPage.jsx

```diff
@@ -1,40 +1,43 @@
-import {useNavigate} from "react-router-dom";
-import {useState} from "react";
-import {loginUser} from "./Auth";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import { loginUser } from "./Auth";
 import "./LoginPage.css";
 
-
 function LoginPage() {
   const navigate = useNavigate();
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
 
-  const handleSignUpRoute = ()=>{
-    navigate('/SignUpPage');
+  const handleSignUpRoute = () => {
+    navigate("/SignUpPage");
   };
-  const handleResetPasswordRoute = ()=>{
-    navigate('/ResetPasswordPage');
+  const handleResetPasswordRoute = () => {
+    navigate("/ResetPasswordPage");
   };
 
   const handleLogin = async (event) => {
     event.preventDefault();
     try {
       const result = await loginUser(email, password);
-      if (result !==undefined) {
+      if (result !== undefined) {
         navigate("/MainPage");
-      } else{
+      } else {
         alert("Incorrect password");
```

Comment:
Mentioned similar idea earlier - you can show this error message next to the password input in the form instead of as an `alert`